Main/Wordle/Wordle/WordleGame.cpp

```diff
@@ -0,0 +1,190 @@
+#include "WordleGame.h"
+#include <iostream>
+#include <fstream>
+#include <ctime>
+#include <unordered_set>
+#include <algorithm>
+
+constexpr uint_fast8_t WORD_LENGTH = 5;
+
+WordleGame::WordleGame(const std::string& filePath) : attempts(0), wordOfTheDayMode(false) {
+    loadDatabase(filePath);
+    srand(static_cast<unsigned int>(time(nullptr)));
+}
+
+void WordleGame::start() {
+    int choice;
+    while (true) {
+        displayMenu();
+        std::cin >> choice;
+
+        if (choice == 0) {
+            break;
+        }
+        else if (choice == 1) {
+            if (isWordOfTheDayGuessed()) {
+                std::cout << "Already found! Come back tomorrow!\n";
+                continue;
+            }
+            targetWord = getWordOfTheDay();
+            wordOfTheDayMode = true;
+        }
+        else if (choice == 2) {
+            targetWord = getRandomWord();
+            wordOfTheDayMode = false;
```

Comment:
I would go with `enum class GameMode {WordOfTheDay, RandomWord }`
It's not clear what is it in case `wordOfTheDay == false`